### ISSUE TITLE
Visual indicator for which station is viewed

### ISSUE THREAD

**Issue by bnhovde** [tommyno/fuglesang#1]
Would be nice to see what station you're viewing. Depends if the station name is part of the api..

**Comment by tommyno** [comment]
Agree! Station name is unfortunately not available in their REST API: https://app.birdweather.com/api/v1

But moving to their new GraphQL API should make this possible:
https://app.birdweather.com/api/#definition-Station
